Commands/Base/PipeBinds/UserPipeBind.cs

```diff
@@ -56,9 +56,9 @@ public User GetUser(ClientContext context)
             {
                 user = User;
             }
-            else if (User.Id > 0)
```

Comment:
This is a typo from a previous refactor. As the first if checks for "User != null", we can reach this line only if User is null.